include/asterisk/rtp_engine.h

```diff
@@ -705,7 +705,7 @@ struct ast_rtp_engine {
 	/*! Callback for retrieving a file descriptor to poll on, not always required */
 	int (*fd)(struct ast_rtp_instance *instance, int rtcp);
 	/*! Callback for initializing RED support */
-	int (*red_init)(struct ast_rtp_instance *instance, int buffer_time, int *payloads, int generations);
+	int (*red_init)(struct ast_rtp_instance *instance, int buffer_time, int payload, int generations);
```

Comment:
This needs to stay a pointer for API compat. I don't see a reason in why this had to change.

Reply:
If I understand it correctly, the existing red functionality was not working anymore after the removal of the chan_sip stack. We refactored the code to use the attribute structure similar to other codecs, as you requested earlier. For this reason the data type inside the structure changed from int array to int. This is the type which is supported from the attribute framework and used for other codecs as well. Therefore the API was changed, as the array don't exists anymore.

Reply:
OK, then this is not eligible for inclusion in any release branches, so you can update the `cherry-pick-to` comment to `cherry-pick-to: none`

Reply:
Asterisk 24 is released in summer, as I understood, so its not that far away. But I will quickly discuss with our customer about this topic.

Reply:
Just going over some older remarks, this was resolved, the API was not changed.